packages/owl-bot/src/handlers.ts

```diff
@@ -301,16 +304,18 @@ export async function refreshConfigs(
   if (stored) {
     logger.info(`Stored new configs for ${repoFull}`);
     for (const badConfig of badConfigs) {
-      await createIssueIfTitleDoesntExist(
-        octokit,
-        githubRepo.owner,
-        githubRepo.repo,
-        badConfig.path + ' is broken.',
-        [
-          'This repo will not receive automatic updates until this issue is fixed.\n',
-          ...badConfig.errorMessages,
-        ].join('\n* ')
-      );
+      if (!CONFIG_PATHS_TO_SKIP.includes(badConfig.path)) {
```

Comment:
When the path is skipped, please write a log statement citing the path that was skipped and why.